src/main/java/com/openlattice/chronicle/services/delete/DataDeletionService.java

```diff
@@ -168,14 +171,16 @@ private void deleteStudyData(
                     .filter( Sets.newHashSet( devicesESID, appDataESID, preprocessedDataESID ), Objects::nonNull );
             Set<UUID> dstEntitySetIds = Sets.filter( Sets.newHashSet( answersESID ), Objects::nonNull );
 
-            deleteParticipantNeighbors(
-                    dataApi,
-                    searchApi,
-                    srcEntitySetIds,
-                    dstEntitySetIds,
-                    participantsToDelete,
-                    participantsESID,
-                    deleteType );
+            executor.execute( () -> {
+                deleteParticipantNeighbors(
+                        dataApi,
+                        searchApi,
+                        srcEntitySetIds,
+                        dstEntitySetIds,
+                        participantsToDelete,
+                        participantsESID,
+                        deleteType );
+            } );
```

Comment:
`deleteParticipantNeighbors` calls `dataApi.deleteEntities` so it makes sense to wrap this. However, there is another call to `dataApi.deleteEntities` below on line 186 and 197, and I think that will also need to be moved to the background.

Reply:
setting `blockUntilCompletion` to `true` in the DataApi.deleteEntities call executes in the background, no? In that case, I don't think there is need wrap this in a background thread